toggle/src/component/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Switch, FormControl, FormLabel } from "@chakra-ui/react";
+import { ReactAsset } from "@player-ui/react";
+import type { TransformedToggle } from "../types";
+
+const useToggleProps = (props: TransformedToggle) => {
```

Comment:
if there is no need this hook we can delete lines 6 - 10 and use `= props` at line 13.